### PR TITLE
Firefox 81 released

### PR DESCRIPTION
**Pull request by afmenez**
Firefox 81 will be released tomorrow, but it is already available on the ftp servers.

### CODE REVIEW

File: environments.json [hunk 1]
```diff
@@ -1269,7 +1269,7 @@
     "short": "FF 60 ESR",
     "release": "2018-05-09",
     "retire": "2019-10-22",
-    "obsolete": "very"
+    "obsolete": true
```
Comment (chicoxyzzy) [inline environments.json]: why?

Comment (afmenez) [reply environments.json]: The "rule" I was using was "1 year after retire date".  This will be true until the next release.

**Review by ljharb — APPROVED** [review]
Assuming the release went out as scheduled.

File: environments.json (hunk 1 above)
Comment (chicoxyzzy) [reply environments.json]: I could be wrong, but the rule was about release dates. Anyway, I don't think we should add 60 ESR to the table

Comment (afmenez) [reply environments.json]: FF60 removed.

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**